test/e2e/probe_whitelist_test.go

```diff
@@ -0,0 +1,83 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"testing"
+
+	pkgTest "knative.dev/pkg/test"
+	"knative.dev/pkg/test/logstream"
+	"knative.dev/serving/test"
+	v1a1test "knative.dev/serving/test/v1alpha1"
+)
+
+//This test checks if the activator can probe
+//the service when istio end user auth policy is
+//applied on the service.
+//This test needs istio side car injected and
+//istio policy check enabled. If both are not
+//enabled the test will pass
+//policy is present test/config/auth/policy.yaml
```

Comment:
`test/config/auth/policy.yaml` -> `test/config/security/policy.yaml`

Reply:
changed. Thank you.